### PR TITLE
Expose dataset generation in CLI

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

File: bin/ota-dataset.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { program } from 'commander';
+import cron from 'croner';
+
+import { release } from '../scripts/dataset/index.js';
+import logger from '../src/logger/index.js';
+
+program
+  .name('ota dataset')
+  .description('Export the versions dataset into a ZIP file and publish it to GitHub releases')
+  .option('-f, --fileName <fileName>', 'file name of the generated dataset')
```
[the remark this answers is not in the document]
Comment (Ndpnt) [reply bin/ota-dataset.js:13]: Indeed, (1) and (2) are true but as it was not exposed in the CLI before, I did not added in the changelog as breaking change.

File: bin/ota-dataset.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  shouldRemoveLocalCopy: removeLocalCopy,
+};
+
+if (!schedule) {
+  await release(options);
+} else {
+  logger.info('The scheduler is running…');
+  logger.info('Dataset will be published at 08:30 on every Monday');
+
+  cron('30 8 * * MON', () => release(options));
```
[the remark this answers is not in the document]
Comment (Ndpnt) [reply bin/ota-dataset.js:32]: Let's do this in another PR

File: bin/ota-dataset.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (MattiSG) [reply bin/ota-dataset.js]: Isn't it breaking that the last argument is not anymore interpreted as the filename?

For example, if I used `ota dataset mydataset.zip`, wouldn't this not work anymore as I now need to update my scripts to `ota dataset --file mydataset.zip? 🙂

**Review by MattiSG — APPROVED** [review]
Great! This changeset is clean. As mentioned in https://github.com/ambanum/OpenTermsArchive/pull/983#discussion_r1048104089, I would really appreciate that we add the snake case for options rule in CONTRIBUTING before merging, to avoid taking governance debt 🙂

**Comment by MattiSG** [comment]
❤️

**Pull request merged**